compiler/noirc_frontend/src/hir/resolution/resolver.rs

```diff
@@ -957,6 +957,25 @@ impl<'a> Resolver<'a> {
                 let stmt = HirAssignStatement { lvalue: identifier, expression };
                 HirStatement::Assign(stmt)
             }
+            Statement::For(for_loop) => {
+                let start_range = self.resolve_expression(for_loop.start_range);
+                let end_range = self.resolve_expression(for_loop.end_range);
+                let (identifier, block) = (for_loop.identifier, for_loop.block);
+
+                // TODO: For loop variables are currently mutable by default since we haven't
```

Comment:
Non-blocking but do we have an issue for this?

Reply:
Ah I see this comment is leftover

Reply:
I don't believe we do